SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -490,8 +487,9 @@ void vtkSlicerSkeletalRepresentationInitializerLogic::AddModelNodeToScene(vtkPol
     scene->AddNode(modelNode);
 
 }
-int vtkSlicerSkeletalRepresentationInitializerLogic::ShowFittingEllipsoid(vtkPoints* points, double curr_volume, double center[3])
+int vtkSlicerSkeletalRepresentationInitializerLogic::ShowFittingEllipsoid(vtkPolyData* mesh)
 {
+    vtkSmartPointer<vtkPoints> points = mesh->GetPoints();
     Eigen::MatrixXd point_matrix(points->GetNumberOfPoints(), 3);
```

Comment:
Just for optimization, you might want to use fixed compile time dimensions for your Eigen3 matrix if you know them.
There is a significance performance gain computing eigenvalues.
```
typedef Matrix<double, Dynamic, 3> MatrixX3d;
```
instead of:
```
typedef Matrix<double, Dynamic, Dynamic> MatrixXd;
```